crates/ruma-client/CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # [unreleased]
 
+Breaking changes:
+
+* Remove `isahc` feature
+
 Improvements:
 
-- Add `error_kind` accessor method to `Error<E, ruma_client_api::Error>`
+* Add `error_kind` accessor method to `Error<E, ruma_client_api::Error>`
```

Comment:
We actually use `-` for unordered list now.

Reply:
Right. Fixed it! The older changelog entries had `*`.